### PR TITLE
the-sz.Banks version 1.02

### PR DESCRIPTION
**Pull request by the-sz**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/90562)

### CODE REVIEW

File: manifests/t/the-sz/Banks/1.02/the-sz.Banks.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+PackageIdentifier: the-sz.Banks
+PackageVersion: 1.02
+PackageLocale: en-US
+Publisher: the sz development
+PublisherUrl: https://the-sz.com/
+PrivacyUrl: https://the-sz.com/products/privacy.php
+Author: the sz development
+PackageName: Banks
+PackageUrl: https://the-sz.com/products/banks/
+License: Copyright (c) the-sz.com
```
Comment (Trenly) [inline manifests/t/the-sz/Banks/1.02/the-sz.Banks.locale.en-US.yaml:13]: A copyright isn't a license; Please update this with actual license terms such as Proprietary, MIT, GPL-2.0, or other terms as appropriate

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by Trenly** [comment]
@msftbot zip

**Comment by ItzLevvie** [comment]
@msftbot Blocking-Issue

**Review by ItzLevvie — DISMISSED** [review]

**Comment by Trenly** [comment]
[Policy] Reset Feedback

File: manifests/t/the-sz/Banks/1.02/the-sz.Banks.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: the-sz.Banks
+PackageVersion: 1.02
+InstallerType: zip
+Installers:
+- Architecture: x86
+  InstallerUrl: https://the-sz.com/common/get.php?product=banks
```
Comment (Trenly) [inline manifests/t/the-sz/Banks/1.02/the-sz.Banks.installer.yaml:9]: Missing NestedInstallerType and NestedInstallerFiles

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/t/the-sz/Banks/1.02/the-sz.Banks.installer.yaml (hunk 2 above)
Comment (the-sz) [reply manifests/t/the-sz/Banks/1.02/the-sz.Banks.installer.yaml:6]: NestedInstallerType and NestedInstallerFiles added

**Review by the-sz — COMMENTED** [review]
NestedInstallerType and NestedInstallerFiles added

**Review by stephengillie — APPROVED** [review]

**Pull request merged**